.github/workflows/delete-model-runs.yaml

```diff
@@ -0,0 +1,58 @@
+# Workflow that can be manually dispatched to delete test model runs that
+# do not need to be persisted indefinitely.
+#
+# Gated such that it's impossible to delete runs older than the current
+# assessment cycle, where each assessment cycle starts in April.
+
+name: delete-model-runs
+
+on:
+  workflow_dispatch:
+    inputs:
+      run-ids:
+        description: >
+          Comma-delimited list of IDs of model runs to delete (no spaces). Note
+          that the workflow assumes these IDs correspond to model runs for the
+          current upcoming assessment cycle, and if that's not the case the
+          deletion script will raise an error.
+        required: true
+        type: string
+
+jobs:
+  delete-model-runs:
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to interact with GitHub's OIDC Token endpoint so we can auth AWS
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout repo code
+        uses: actions/checkout@v4
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install system dependencies
+        run: sudo apt-get install libgit2-dev
+        shell: bash
+
+      - name: Disable renv sandbox to speed up install time
+        run: echo "RENV_CONFIG_SANDBOX_ENABLED=FALSE" >> .Renviron
+        shell: bash
+
+      - name: Setup renv
+        uses: r-lib/actions/setup-renv@v2
```

Comment:
Taken together, these four steps (from `Setup R` to `Setup renv`) install the dependencies necessary to run the script. R and renv packages are cached based on the R version and `renv.lock`. In total, the four steps comprising dependency installation took 43s to complete in [the most recent successful run](https://github.com/ccao-data/model-res-avm/actions/runs/6908697011/job/18798517688).

One alternative I considered was to pull the Docker image for the current branch saved in GHCR and use that to run the deletion script. While this might be slightly faster than installing dependencies on every run, I decided against it because it felt like it would introduce a fragile implicit dependency between this workflow and the `build-and-run-model` workflow, the maintenance burden of which seemed to me to outweigh the convenience of saving a few seconds on install time. But I'm open to pushback!

Reply:
No I think you made the correct choice, it's totally sensible to just use renv + cache here.